setup.py

```diff
@@ -63,6 +63,7 @@
     python_requires='>=' + python_requires_str,
     install_requires=[
         'requests>=2.19,<3',
+        'httpx=0.11.*',
```

Comment:
Hi, just read #133 and the async implementation idea seems really interesting.

This might be an absurd question because I'm not too well informed about httpx and async, but why don't you use it for both sync and async requests? The official httpx repo indicates that it works for both.

Reply:
I outline some reasons in #131, let me know what you think!